### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by kingqabil**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -180,6 +180,24 @@ workProjects.forEach((project, index) => {
 
 projectsContainer.innerHTML = projectHTML;
 
+const popupWindow = () => {
+  show([popupContainer]);
+  // body.style.overflow = 'hidden';
```
Comment (carlos-ssh) [inline script.js]: - Remove commented code.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # Portfolio: setup and mobile version skeleton
 > My Eighty  Project Portfolio - mobile menu.
 
-![screenshot](./imgs/app-image.png)
+![screenshot](./imgs/popup.jpg)
```
Comment (carlos-ssh) [inline README.md:7]: Keep professionalism in your projects. 
- Kindly provide a description about the features implemented in your Pull Request.

File: script.js [hunk 3]
```diff
@@ -180,6 +180,24 @@ workProjects.forEach((project, index) => {
...
+const closePopupWindow = () => {
+  hide([popupContainer]);
+  body.style.overflow = 'scroll';
+  blurElements([headlineSection, portfolioSection, aboutSection, contactSection], false);
+};
 
+// close the popup modal
+closePopupButton.addEventListener('click', closePopupWindow);
 
+// iterate through the items and listen for popupwindow click
```
Comment (carlos-ssh) [inline script.js]: [OPTIONAL]
- Remove commented code. Your code explains by itself what it do.

File: script.js [hunk 4]
```diff
@@ -180,6 +180,24 @@ workProjects.forEach((project, index) => {
```
Comment (carlos-ssh) [inline script.js:180]: - Provide the requested styles guidelines in your Figma template.
**Requested style:**
Screen Shot 2021-10-12 at 10 42 55

#

**Yours:**
Screen Shot 2021-10-12 at 10 46 34

**_ Kindly check your paddings_**

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @kingqabil , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

changesrequested


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 4 above)
Comment (GzimAsani) [reply script.js:180]: Please improve this section in order to match the figma project!
Screenshot from 2021-10-12 22-20-18

File: README.md (hunk 2 above)
Comment (GzimAsani) [reply README.md:7]: - [ ] Please implement this requirement asked by the previous reviewer

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Sadqhabil         
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good use of gitflow :+1: 

## Required Changes ♻️
- [ ] PR description should be used to write the changes that you've made in this pull request and to explain your work. Please add a more descriptive PR


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @kingqabil & @Lockless404, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**